.github/workflows/build_and_test.yml

```diff
@@ -71,17 +73,37 @@ jobs:
              export IPDK_RECIPE=`pwd`
              cd $IPDK_RECIPE/setup
              cmake -B build -DCMAKE_INSTALL_PREFIX=${GITHUB_WORKSPACE}/deps
-             cmake --build build -j6
+             cmake --build build -j6   
```

Comment:
remove extra space?

Reply:
As a matter of practice, we should trim trailing white space before committing the file.

Reply:
Yep, i updated my vscode settings!